dspace/config/crosswalks/oai/xoai.xml

```diff
@@ -70,6 +69,28 @@
                 This contexts complies with OpenAIRE Guidelines for Literature Repositories v3.0.
             </Description>
         </Context>
+
+        <!--
+            OpenAIRE Guidelines 4.0:
+
+            - https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/
+
+            There is a limitation over the embargoedEndDate parameter:
+
+            - Predefined DSpace fields don't allow to set this up with a default.
+         -->
+        <Context baseurl="openaire4" name="OpenAIRE 4 Context">
+            <!-- Date format, field prefixes, etc are ensured by the transformer -->
+            <!--Transformer ref="openaire4Transformer"/-->
```

Comment:
Is there a reason why the transformer is commented out by default?

Reply:
Until the transformer be ready there as a reason but currently there is no reason. Thank you @tdonohue for spotting this!